src/rabbit_control_main.erl

```diff
@@ -519,6 +519,9 @@ action(list_vhosts, Node, Args, _Opts, Inform, Timeout) ->
     display_info_list(call(Node, {rabbit_vhost, info_all, []}, Timeout),
                       ArgAtoms);
 
+action(list_user_permissions, _Node, _Args = [], _Opts, _Inform, _Timeout) ->
```

Comment:
Yes, if we want to handle empty values, it needs to be a separate issue (not sure how many run into that).